### PR TITLE
People 154 generate user skill rates

### PR DESCRIPTION
**Pull request by DariuszMusielak**
Ticket
https://netguru.atlassian.net/browse/PEOPLE-154

# Description
Added rake task for generating user skill rates for all of the users with default values

screen shot 2017-01-20 at 13 03 36

### CODE REVIEW

File: lib/tasks/user_skill_rates.rake [hunk 1]
```diff
@@ -0,0 +1,17 @@
+namespace :user_skill_rates do
+  desc 'Generate user_skill_rates with default values for all users'
+  task generate_all: :environment do
+
+    active_users =  User.active
```
Comment (netguru-hound) [inline lib/tasks/user_skill_rates.rake]: Operator = should be surrounded by a single space.<br>Unnecessary spacing detected.

File: lib/tasks/user_skill_rates.rake [hunk 2]
```diff
@@ -0,0 +1,17 @@
+namespace :user_skill_rates do
+  desc 'Generate user_skill_rates with default values for all users'
+  task generate_all: :environment do
+
```
Comment (netguru-hound) [inline lib/tasks/user_skill_rates.rake]: Extra empty line detected at block body beginning.

**Comment by marcinjak** [comment]
BTW: Circle failed :)

**Comment by DariuszMusielak** [comment]
Yep, we have problem with random specs failing. I have added ticket for that.

**Pull request merged**